requirements.txt

```diff
@@ -1,23 +1,22 @@
 boto==2.49.0
 dj-database-url==0.4.2
 django-admin-tools==0.8.1
-django-bleach==0.6.1 
+django-bleach==0.5.3 
 django-ckeditor==5.2.2
-django-envelope==1.0
+django-envelope==1.3
 django-extensions==1.8.1
 django-flatblocks==0.9.3
 django-flatpages-tinymce==0.1.1
 django-forms-bootstrap==2.0.3.post1
-django-grappelli==2.5.6
+django-grappelli==2.8.3
 django-gravatar2==1.1.4
 django-honeypot==0.5.0
 django-ical==1.4
 django-nocaptcha-recaptcha==0.0.20
-django-pgsql-interval-field==0.9.4
 django-storages==1.1.8
```

Comment:
Deleting this lib is needed for the 1.9 upgrade, but it hoses the migrations for a new, clean build.  I'll have to look into squashing this and/or adjusting the migrations manually to deal with this.  We should not merge this into production at this time.